defaults/main/sshd.yml

```diff
@@ -43,17 +47,16 @@ sshd_ignore_user_known_hosts: "yes"
 sshd_kerberos_authentication: "no"
 sshd_kex_algorithms: >-
   curve25519-sha256@libssh.org,
-  ecdh-sha2-nistp521,
-  ecdh-sha2-nistp384,
-  ecdh-sha2-nistp256,
```

Comment:
From the Secure Secure Shell article on NIST curves: "They leak secrets through timing side channels and off-curve inputs. Also, [NIST is considered harmful](https://projectbullrun.org/dual-ec/vulnerability.html) and cannot be trusted."